myapp/spec/system/tasks_spec.rb

```diff
@@ -234,7 +234,7 @@
       expect(page).to have_selector("input[type='radio'][value='in_progress']")
       expect(page).to have_selector("input[type='radio'][value='completed']")
 
-      expect(page).to have_button('Submit Task')
+      expect(page).to have_button('Create Task')
```

Comment:
May be only "Create" is more than okay here.
```suggestion
      expect(page).to have_button('Create')
```

Reply:
Fixed it at 726d7d4